pkg/upstreamyaml/upstreamyaml.go

```diff
@@ -33,9 +33,15 @@ type UpstreamYaml struct {
 	HelmRepo           string         `json:"HelmRepo,omitempty"`
 	Hidden             bool           `json:"Hidden,omitempty"`
 	Namespace          string         `json:"Namespace,omitempty"`
-	PackageVersion     int            `json:"PackageVersion,omitempty"`
-	ReleaseName        string         `json:"ReleaseName,omitempty"`
-	Vendor             string         `json:"Vendor,omitempty"`
+	// PackageVersion is deprecated. It is deprecated because rancher/partner-charts
```

Comment:
Deprecation should follow https://go.dev/wiki/Deprecated like this:
 
// PackageVersion <what is does>
// Deprecated: <why it's deprecated>

Reply:
Interesting, I wasn't aware of that. I'll make that change.